patches/busybox/feature-onie-syseeprom-command.patch

```diff
@@ -58,6 +35,27 @@ index 0000000..c304273
 +#define EEPROM_TYPE_8BIT_ADDR	1
 +#define EEPROM_TYPE_16BIT_ADDR 	2
 +
++/*
++ * SYS_EEPROM_I2C_WRITE_1B_DELAY: delay number of microseconds after writing 1 byte
++ * SYS_EEPROM_I2C_WRITE_2B_DELAY: delay number of microseconds after writing 2 bytes
++ * SYS_EEPROM_I2C_WRITE_3B_DELAY: delay number of microseconds after writing 3 bytes
++ */
++#if CONFIG_SYS_EEPROM_I2C_WRITE_1B_DELAY == 0
++  #define SYS_EEPROM_I2C_WRITE_1B_DELAY   10
++#else
++  #define SYS_EEPROM_I2C_WRITE_1B_DELAY   CONFIG_SYS_EEPROM_I2C_WRITE_1B_DELAY
++#endif
```

Comment:
Is the check for 0 necessary?  The miscutils/Config.src file (down below) defines the default already.

The only way it could be zero is if the user explicitly set it to zero.